lib/codegen/meta-python/base/legalize.py

```diff
@@ -382,6 +401,115 @@
             a << bxor(x, y)
         ))
 
+# Expand bitrev
+# Adapted from Stack Overflow.
+# https://stackoverflow.com/questions/746171/most-efficient-algorithm-for-bit-reversal-from-msb-lsb-to-lsb-msb-in-c
+expand.legalize(
+        a << bitrev8(x),
+        Rtl(
+            a1 << band_imm(x, 0xaa),
```

Comment:
The DSL needs these immediates to be wrapped in `imm64`, like `imm64(0xaa)`.

Reply:
Whoops, I knew that and forgot. That must be what's wrong. 😅